tests/commands/test_list.py

```diff
@@ -78,11 +78,11 @@ async def test_list_publications_from_window(capsys, generate_models):
 async def test_list_publications_to_window(capsys, generate_models):
     await generate_models(spec)
     await list_publications(
-        to=arrow.Arrow(year=2021, month=6, day=6, hour=3, minute=30)
```

Comment:
maybe the problem is here: why changing the hour at all?

Reply:
same